python/cugraph/cugraph/tests/docs/test_doctests.py

```diff
@@ -186,11 +186,4 @@ def test_docstring(self, docstring):
                 f"{docstring.name}:\n{doctest_stdout.getvalue()}"
             )
         except AssertionError:
```

Comment:
Because this is no longer using specialized logic in the exception clause, the "except: raise" pattern is unneeded. We can remove the `try` entirely.